### ISSUE TITLE
HTML is not very semantic

### ISSUE THREAD

**Issue by DaemonSpelledWrong** [DaemonSpelledWrong/self-portfolio#42]
Not everything should be a section. Update a lot of the section tags to something more semantic for user accessibility.
